Kernel/Process.cpp

```diff
@@ -2929,7 +2931,7 @@ int Process::sys$mount(const char* device_path, const char* mountpoint, const ch
 
     auto& mountpoint_custody = custody_or_error.value();
 
-    RefPtr<FS> fs { nullptr };
+    RefPtr<FS> fs{ nullptr };
```

Comment:
This alignment is not consistent with the coding style. Are you using an old version of `clang-format` perhaps?

Reply:
Yep that's what it looks like. Default clang-format on Ubuntu-18.04 was clang-format-6.0. clang-format-8.0 seems to like the style better.